### PR TITLE
fix(style): fix style error message for ValidFieldTextArea #patch

### PR DESCRIPTION
**Pull request by andrinuryawan**
fix style error message for ValidFieldTextArea
before
image

after
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,149 +4,217 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.7.3]
```
Comment (rayhan-ab) [inline CHANGELOG.md:7]: i guess you should turn your autoformatter off before changing changelog

File: src/components/ValidatedInput/ValidFieldTextArea.scss [hunk 2]
```diff
@@ -15,6 +15,9 @@
     font-size: $font-size-h6;
     padding-bottom: 4px;
   }
+  .field-error-message {
+    font-size: $font-size-h7;
```
Comment (rayhan-ab) [inline src/components/ValidatedInput/ValidFieldTextArea.scss:19]: if we only change it here, there will be inconsistency among error message in different input components.

**Comment by rayhan-ab** [comment]
please run `yarn build` before pushing the commit to make sure the build is up to date before merging to master

**Comment by rayhan-ab** [comment]
iirc, there is also inconsistency in `FieldLabel` font size. how about setting it to $font-size-h6 in Form/utility.scss because `FieldLabel` in `ValidatedInput` components are 14px.
note: we keep `FieldLabel` in `TableFilter` 16px
wdyt?

**Comment by andrinuryawan** [comment]
> iirc, there is also inconsistency in `FieldLabel` font size. how about setting it to $font-size-h6 in Form/utility.scss because `FieldLabel` in `ValidatedInput` components are 14px. note: we keep `FieldLabel` in `TableFilter` 16px wdyt?

ok i wil set it
ty

**Review by rayhan-ab — APPROVED** [review]

**Pull request merged**